### ISSUE TITLE
GPL-nnn As a long read user I would like the library kit barcode to be the first or last field

### ISSUE THREAD

**Issue by stevieing** [sanger/traction-ui#512]
**User story**
As a long read user I would like the library kit barcode to be the first or last field when I create a library

**Who are the primary contacts for this story**
Craig

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] When I go to the create library page the library kit barcode should be either the first or last field on the page.
- [ ] ...

**Dependencies**
This story is blocked by the following dependencies:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

**Additional context**

**Comment by stevieing** [comment]
No longer relevant as we now use the pooling page.
